src/Resources/views/Ckeditor/browser.html.twig

```diff
@@ -77,57 +77,63 @@ file that was distributed with this source code.
 
     <div class="col-xs-12 col-md-9">
         <div class="box box-primary">
-            <div class="box-body {% if admin.datagrid.results|length > 0 %}table-responsive no-padding{% endif %}">
+            <div class="box-body {% if datagrid.results|length > 0 %}table-responsive no-padding{% endif %}">
                 {{ sonata_block_render_event('sonata.admin.list.table.top', { 'admin': admin }) }}
 
                 {% block list_header %}{% endblock %}
 
-                {% if admin.datagrid.results|length > 0 %}
+                {% if datagrid.results|length > 0 %}
```

Comment:
same

Reply:
Yep - basically this will not work with ORM admin, but since you pass datagrid directly why not use it this way?